api/envoy/config/cluster/v3/cluster.proto

```diff
@@ -635,6 +635,15 @@ message Cluster {
     //   The specified percent will be truncated to the nearest 1%.
     type.v3.Percent healthy_panic_threshold = 1;
 
+    // Zone aware load balancing configuration.
+    //
+    // .. note::
+    //
+    //   This field may be deprecated in the future in favor of the :ref:`load_balancing_policy
```

Comment:
Because I hesitated about to deprecate all these APIs. They are used by every Envoy users in every scenario. And I think most users may only set a simple Enum (`lb_policy` field) or even don't set anything to use the default Enum to use the default load balancer (round robin).

Deprecate this APIs make no other sense except will create a bunch of noisy. (we need maintain these APIs forever anyway.)

I will remove these comment after a rethinking about that.